app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -1,13 +1,15 @@
 class ManageIQ::Providers::Openshift::ContainerManager < ManageIQ::Providers::ContainerManager
   include ManageIQ::Providers::Openshift::ContainerManagerMixin
 
+  require_nested :ContainerImage
```

Comment:
:+1: I took a quick look and I think we need to fix this problem in a couple of places.
missing require_nested. is there some criteria to the files we need to require or do we need to require everything?

I think we should probably make some names absolute too (`::Container`). wdyt @cben ?